torchci/pages/minihud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -231,12 +234,161 @@ function CommitSummaryLine({
           </a>
         </span>
       )}
+      {ttsAlert && (
+        <span style={{ float: "right" }}>
+          <b>tts alert</b>
+        </span>
+      )}
       <CommitLinks row={row} />
     </div>
   );
 }
 
-function CommitSummary({ row }: { row: RowData }) {
+function DurationInfo({
+  jobs,
+  prevRow,
+  expandAllDurationInfo,
+}: {
+  jobs: JobData[];
+  prevRow: RowData | undefined;
+  expandAllDurationInfo: boolean;
+}) {
+  function getAggregateTestTimes(jobs: JobData[] | undefined) {
+    return _.reduce(
+      jobs,
+      (
+        prev: {
+          [key: string]: {
+            duration: number;
+            validData: boolean;
+          };
+        },
+        cur
+      ) => {
+        // filter out rocm and macos because they tend to be variable
+        if (
+          cur.name != undefined &&
+          cur.name.includes(",") &&
```

Comment:
I'm guessing it's to exclude stuff like "Create Release"?

Reply:
its because most of our tests include something like (test_config, shard number, ... but if there are other jobs we want to consider, i can change that